src/app/pages/ConsensusAccountDetailsPage/ConsensusAccountDetailsCard.tsx

```diff
@@ -13,18 +13,25 @@ type ConsensusAccountDetailsCardProps = {
   account: Account | undefined
   isError: boolean
   isLoading: boolean
+  highlightedPartOfName?: string | undefined
 }
 
 export const ConsensusAccountDetailsCard: FC<ConsensusAccountDetailsCardProps> = ({
   account,
   isError,
   isLoading,
+  highlightedPartOfName,
 }) => {
   const { t } = useTranslation()
 
   return (
     <SubPageCard featured isLoadingTitle={isLoading} title={t('account.title')} mainTitle>
-      <ConsensusAccountDetailsView isError={isError} isLoading={isLoading} account={account} />
+      <ConsensusAccountDetailsView
+        isError={isError}
+        isLoading={isLoading}
```

Comment:
Something isn't handling loading state correctly:
reload [an account](https://92fdad12.oasis-explorer.pages.dev/mainnet/consensus/address/oasis1qp000ff4weh6t6083hazdpw0r902leffp5gjgu55) a few times to see an warning before it gets loaded
[screen-recording.webm](https://github.com/oasisprotocol/explorer/assets/3758846/839e3302-9df5-4828-8fa8-b8f7cf084a73)

Reply:
I can reproduce the problem, but unfortunately this is also happening on master branch (at least since merging #1422), so the issue is independent of the current PR. (I will fix this in a follow-up PR.)

Reply:
See fix in #1423.